include/fastdds/dds/xtypes/dynamic_types/DynamicTypeMember.hpp

```diff
@@ -87,7 +87,7 @@ class DynamicTypeMember : public std::enable_shared_from_this<DynamicTypeMember>
 
     /**
      * State comparison according with the [standard] sections \b 7.5.2.6.3
-     * @param[in] other @DynamicTypeMember reference to compare to
+     * @param[in] other DynamicTypeMember reference to compare to
```

Comment:
```suggestion
     * @param[in] other @ref DynamicTypeMember reference to compare to
```

Reply:
Fixed in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)